guides/micronaut-security-basicauth/groovy/src/main/groovy/example/micronaut/DefaultAuthorizationExceptionHandlerReplacement.groovy

```diff
@@ -3,22 +3,27 @@ package example.micronaut
 import io.micronaut.context.annotation.Replaces
 import io.micronaut.http.HttpRequest
 import io.micronaut.http.HttpResponse
-import io.micronaut.http.HttpStatus
 import io.micronaut.http.MutableHttpResponse
 import io.micronaut.security.authentication.AuthorizationException
 import io.micronaut.security.authentication.DefaultAuthorizationExceptionHandler
 import jakarta.inject.Singleton
 
+import static io.micronaut.http.HttpHeaders.WWW_AUTHENTICATE
+import static io.micronaut.http.HttpStatus.FORBIDDEN
+import static io.micronaut.http.HttpStatus.UNAUTHORIZED
+
 @Singleton // <1>
 @Replaces(DefaultAuthorizationExceptionHandler) // <2>
 class DefaultAuthorizationExceptionHandlerReplacement extends DefaultAuthorizationExceptionHandler {
 
     @Override
-    protected MutableHttpResponse<?> httpResponseWithStatus(HttpRequest request, AuthorizationException exception) {
-        if (exception.isForbidden()) {
-            return HttpResponse.status(HttpStatus.FORBIDDEN)
+    protected MutableHttpResponse<?> httpResponseWithStatus(HttpRequest request,
+                                                            AuthorizationException e) {
+        if (e.isForbidden()) {
```

Comment:
I like the original flow without an else bracket better. I think it is nicer to only remove the return statement when it is the last sentence.

Reply:
updated